html/semantics/embedded-content/the-iframe-element/support/download_stash.py

```diff
@@ -23,6 +23,6 @@ def main(request, response):
     for i in range(count): # pylint: disable=unused-variable
         time.sleep(single_delay)
         response.writer.write_content(u"\n")
-        if not response.writer.flush():
+        if not response.writer.write(b"."):
```

Comment:
This one is a bit more problematic. We need to change `write_content` (and correspondingly `write_raw_content` and `write_content_file`) to surface the return value so that we can check here, to avoid adding more stuff to the response than the test author wanted.

Reply:
Code updated. Thank you!